sectionsUtilityTool/src/main/resources/sectiontool.properties

```diff
@@ -0,0 +1,35 @@
+#/api/v1/accounts/1/terms?per_page=4000&_=1426266199833
+# ^ starting of exp. $ ending of exp
+# ? is a quantifier in regex. hence escaping
+# GET request
+canvas.api.terms.regex=^/api/v1/accounts/1/terms\\?per_page=\\d+&_=\\d+$
```

Comment:
I did not write any junit test cases. But i have tested with the Real data from canvas environments